process.ipynb

```diff
@@ -36,6 +24,7 @@
   {
    "cell_type": "code",
    "execution_count": null,
+   "id": "ea477d94",
```

Comment:
Wonder what's `id`?

Reply:
https://jupyter.org/enhancement-proposals/62-cell-id/cell-id.html

Seems it's just an id for the cell, it doesn't do anything other than make it unique